### PR TITLE
Add flags to disable video recording and performance metrics

### PR DESCRIPTION
**Pull request by sridoodla**

### CODE REVIEW

File: Flank/src/main/java/com/walmart/otto/tools/GcloudTool.java [hunk 1]
```diff
@@ -37,6 +38,7 @@ public void runGcloud(String testCase, String bucket, int shardIndex)
           "instrumentation",
           orchestratorFlag(),
           autoGoogleLoginFlag(),
+          recordVideoFlag(),
```
Comment (bootstraponline) [inline Flank/src/main/java/com/walmart/otto/tools/GcloudTool.java]: performanceMetricsFlag() isn't used?

Comment (sridoodla) [reply Flank/src/main/java/com/walmart/otto/tools/GcloudTool.java]: 🤦‍♂️  Will fix.

File: Flank/src/main/java/com/walmart/otto/configurator/Configurator.java [hunk 2]
```diff
@@ -201,6 +204,22 @@ public void setDisableAutoGoogleLogin(boolean disableAutoGoogleLogin) {
     this.disableAutoGoogleLogin = disableAutoGoogleLogin;
   }
 
+  public boolean recordVideo() {
+    return recordVideo;
+  }
+
+  public void setRecordVideo(boolean shouldRecordVideo) {
+    recordVideo = shouldRecordVideo;
```
Comment (bootstraponline) [inline Flank/src/main/java/com/walmart/otto/configurator/Configurator.java]: the pattern seems to be:

`this.recordVideo = recordVideo`

It'd be nice to keep the code style consistent.

**Comment by bootstraponline** [comment]
One small comment, otherwise looks good.

**Pull request merged**